collect_app/src/main/java/org/odk/collect/android/activities/FirstLaunchActivity.kt

```diff
@@ -72,12 +76,17 @@ class FirstLaunchActivity : LocalizedActivity() {
                 setOnClickListener {
                     Analytics.log(AnalyticsEvents.TRY_DEMO)
 
-                    projectsRepository.save(Project.DEMO_PROJECT)
-                    projectsDataService.setCurrentProject(Project.DEMO_PROJECT_ID)
-
-                    ActivityUtils.startActivityAndCloseAllOthers(
-                        this@FirstLaunchActivity,
-                        MainMenuActivity::class.java
+                    scheduler.immediate(
```

Comment:
We probably want to throw a progress dialog the shows when this work is happening. `MaterialProgressDialogFragment.showOn` with this code behind a ViewModel would be a good way to go. Otherwise, the user will be able to click on other buttons while the background work executes.

Reply:
Done.

Reply:
Ah right, done.